forge-gui/res/cardsfolder/upcoming/chaos_is_my_plaything.txt

```diff
@@ -0,0 +1,9 @@
+Name:Chaos Is My Plaything
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, it deals damage equal to your commander's mana value to each noncommander creature your opponents control.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Permanent.OppCtrl | TgtPrompt$ Select up to one target creature each opponent controls | TargetMin$ 0 | TargetMax$ OneEach | TargetsForEachPlayer$ True | SubAbility$ DBRepeatEach
+SVar:DBRepeatEach:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBDigUntil

```

Comment:
this would lead to each Player digging once for each Player

but I don't think RepeatEach is needed anyway

Reply:
Hm, so what should we do?

Reply:
just get rid of L6...